### DISCUSSION TITLE
Run locally

### DISCUSSION THREAD

**Discussion by Rimapus** [vilhelmprytz/pterodactyl-installer#175] [Q&A]
Hello,
I want to run pterodactyl on "localhost" but when i put "localhost" as hostname he say "cannot connect to the bus, host is down".

If someone have the answer, please write me :)

**Comment by vilhelmprytz** [comment]
Are you trying the script on WSL?

**Reply by Rimapus** [reply]
I installed ubuntu wsl on windows so, yeah

**Reply by vilhelmprytz** [reply]
The script won't work on WSL. You can only install Pterodactyl on a standard Linux machine.

**Reply by Rimapus** [reply]
oh... ok thank you :)

**Reply by sinjs** [reply]
> 
> 
> The script won't work on WSL. You can only install Pterodactyl on a standard Linux machine.

Technically it can work on WSL 2

**Reply by vilhelmprytz** [reply]
> > The script won't work on WSL. You can only install Pterodactyl on a standard Linux machine.
> 
> Technically it can work on WSL 2

Does it? While you can run Docker and such, would you be able to get a web server running in the background through WSL?

**Reply by sinjs** [reply]
> > > The script won't work on WSL. You can only install Pterodactyl on a standard Linux machine.
> > 
> > 
> > Technically it can work on WSL 2
> 
> Does it? While you can run Docker and such, would you be able to get a web server running in the background through WSL?

I can test it out, I'm talking about WSL v2 and not the by default installed WSL 1.
WSL 2 should include the Linux kernel.

**Reply by vilhelmprytz** [reply]
I understand and I know the difference. WSL 2 is using Hyper-V so perhaps you can get it to run in the background (if you can get systemd or any alternative to that running) but I doubt it will be a smooth solution :slightly_smiling_face:
